### PR TITLE
feat(WebexMember): display icon when member is muted

### PR DESCRIPTION
**Pull request by alinasuciu**
Please give us a feedback for all the changes we added or an overall feedback
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-154242

### CODE REVIEW

File: src/components/WebexMember/WebexMember.jsx [hunk 1]
```diff
@@ -13,22 +13,25 @@ import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
  *
  * @returns {object} JSX of the component
  */
-export default function WebexMember({personID, displayStatus}) {
+export default function WebexMember({personID, displayStatus, isMuted}) {
```
Comment (lalli-flores) [inline src/components/WebexMember/WebexMember.jsx:16]: Passing `isMuted` as a prop to `WebexMember` does not align with the component system design.
In the component system, each component should be as "self contained" as possible, requiring minimal props. In the design, each component is passed an ID of the resource it presents, and with that ID the component (through 
 a hook that wires in the adapter) fetches the data it needs and listens to changes on that resource to update its UI accordingly.
In this case, instead of passing the mute status as a prop, you need to use the membership hook to get that information about this member.

File: src/components/WebexMember/WebexMember.stories.js [hunk 2]
```diff
@@ -18,3 +18,9 @@ StatusEnabled.args = {
   personID: 'user1',
   displayStatus: true,
 };
+
+export const MicrofonMuted = Template.bind({});
```
Comment (lalli-flores) [inline src/components/WebexMember/WebexMember.stories.js:22]: Simplifying the name:
```suggestion
export const Muted = Template.bind({});
```

File: src/adapters/MembershipJSONAdapter.test.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import memberships from '../data/memberships';
```
Comment (lalli-flores) [inline src/adapters/MembershipJSONAdapter.test.js:1]: Thanks for adding missing tests! 🙂
I think you followed the format on the people adapter tests, which is a great start, but those tests where written very early on. Using the [`MeetingsJSONAdapter` tests](https://github.com/webex/components/blob/master/src/adapters/MeetingsJSONAdapter.test.js) is a better guide of where we want to take the tests, in terms of what we test and how we word our test descriptions

**Review by lalli-flores — CHANGES_REQUESTED** [review]
Thank you for putting out a draft PR so quickly! Let me know if any of my feedback is not clear.

On your commits, try to squash them if they are making changes for the same feature. One commit == one feature change, as the commits translate to our changelog items. In terms of structure, we usually add a scope to denote where the change is happening, for instance, this change would be `feat(WebexMember):` as we are adding a new feature to the `WebexMember` component. This way, the change description can be shorter as you don't have to specify in that statement where the change happened (we already know from the scope)

File: src/components/WebexMember/WebexMember.jsx [hunk 4]
```diff
@@ -10,23 +10,38 @@ import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
...
  */
-export default function WebexMember({personID, displayStatus}) {
+export default function WebexMember({
+  personID,
+  displayStatus,
+  destinationID,
+  destinationType,
+}) {
   const {firstName, lastName} = usePerson(personID);
+  const memberships = useMemberships(destinationID, destinationType);
```
Comment (cipak) [inline src/components/WebexMember/WebexMember.jsx:25]: actually, `members`, not `memberships`. The hook should also be renamed to `useMembers`, but maybe not part of this PR.

File: src/components/WebexMember/WebexMember.scss [hunk 5]
```diff
@@ -10,4 +11,12 @@
     text-overflow: ellipsis;
     padding-left: 0.25rem;
   }
+
+  .member-muted {
+    flex: 1;
+    text-align: right;
+    .icon {
+      color: #ff0000;
```
Comment (cipak) [inline src/components/WebexMember/WebexMember.scss:19]: color can be moved up a level

**Review by lalli-flores — COMMENTED** [review]
@alinasuciu, can you make sure that the [Storybook stories](https://github.com/webex/components/blob/master/src/components/WebexMember/WebexMember.stories.js) are updated appropriately? 
You will also need to add a new story to showcase a participant muted

File: src/components/WebexMember/WebexMember.stories.js [hunk 6]
```diff
@@ -18,3 +18,10 @@ StatusEnabled.args = {
   personID: 'user1',
   displayStatus: true,
 };
+
+export const MemberMuted = Template.bind({});
```
Comment (lalli-flores) [inline src/components/WebexMember/WebexMember.stories.js:22]:
```suggestion
export const Muted = Template.bind({});
```
Naming this story `muted` should be sufficient, it will display under the `WebexMember` group under Storybook

**Review by lalli-flores — COMMENTED** [review]
Minor comment, but otherwise good to go!

File: src/components/WebexMember/__snapshots__/WebexMember.stories.storyshot [hunk 7]
```diff
@@ -1,5 +1,115 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Storyshots Platform/Webex Member Member Muted 1`] = `
```
Comment (lalli-flores) [inline src/components/WebexMember/__snapshots__/WebexMember.stories.storyshot:3]: Don't forget to run update snapshot to remove stale snapshots.
```
npm run test -- --updateSnapshot

**Review by lalli-flores — APPROVED** [review]
